macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNAppEventsHandler.swift

```diff
@@ -77,7 +106,16 @@ final class VPNAppEventsHandler {
                     restartLoginItem(using: loginItemsManager)
                 }
             case .some(false) where loginItemsManager.isAnyInstalled(LoginItemsManager.vpnLoginItems):
-                disableLoginItem(using: loginItemsManager)
+                ipcClient.stop { error in
+                    if let error {
+                        self.pixelKit?.fire(LoginItemsControlCheckpointPixel.cannotStopVPN(error))
+                    }
+
+                    Task {
+                        try await Task.sleep(interval: .seconds(2))
+                        self.disableLoginItem(using: loginItemsManager)
+                    }
+                }
```

Comment:
This is the core of the change, instead of just disabling the login item we first stop the VPN.

Reply:
That's not related to these changes - the same behavior is present in `main` and should be present for AuthV2 as well.

DEBUG builds always relaunch the VPN to ensure you're on the latest login item and VPN extension.